meta/main.yml

```diff
@@ -1,16 +1,17 @@
 ---
 galaxy_info:
-  #role_name: dell_powerconnect_switch
+  role_name: dell_powerconnect_switch
```

Comment:
Previously we were able to continue using names with dashes. Is it no longer possible?

Reply:
Ahh, so that was the idea behind removing role_name.
Well ansible-lint was throwing exceptions if role_name was hashed out.
Cant we just change it to dashes ?

Reply:
If you include role_name then it needs to match the official naming scheme which prevents dashes. If you omit it then it should continue to use the old naming scheme when it was first imported.

I don't know if the move to NG galaxy will have changed anything here.

I don't mind too much for these roles since they're only used by kayobe but it would be a shame to have to rename all our other roles.

Reply:
We're already doing it like this in 
https://github.com/stackhpc/ansible-role-systemd-networkd/commit/3ec2354ef6a5c71689dec97fa374a89e75558e7b

(The names are already different in galaxy vs github).

Lets decide on the standard - I can help out and implement across the existing roles. 
I would love for our roles to pass basic lint tests without issues and opinionated skip-lists.

Reply:
Well the standard is to use underscores, so newer roles should have underscores. But that was not required when many of our roles were written. It would be disruptive to rename most of our roles and could break some existing deployments.

Reply:
If NG galaxy will let us, let's try to keep the current names.